### ISSUE TITLE
Teaspeak protocol dosent work

### ISSUE THREAD

**Issue by farhanhyper** [tltneon/lgsl#222]
Hi

I have a teaspeak server and when I put the information of that in the lgsl in says "No Response"

**Comment by wohahobg** [comment]
Hi,

@farhanhyper does your TS server is using different port then default one ?
Does your port is open for public ?

I have same issue with this kind of servers and I have noted that if the port is not the default one it does not return any information only the name (time to time)

**Comment by farhanhyper** [comment]
@wohahobg 
Yes,its running on default port 10101 (teaspeak)
And the server firewall ia disable so its port is open,too

**Comment by wohahobg** [comment]
Yes same issue from my side to any TeamSpeak server. I am thinking is more likely IP banned or so. I am really not sure.
